src/cloud-code/Parse.Cloud.js

```diff
@@ -528,6 +529,37 @@ ParseCloud.beforeConnect = function (handler, validationHandler) {
   );
 };
 
+/**
+ * Sends an email through the Parse Server mail adapter.
+ *
+ * **Available in Cloud Code only.**
+ * **Requires a mail adapter to be configured for Parse Server.**
+ *
+ * ```
+ * Parse.Cloud.sendEmail({
+ *   from: 'Example <test@example.com>',
+ *   to: 'contact@example.com',
+ *   subject: 'Test email',
+ *   text: 'This email is a test.'
+ * });
+ *```
+ *
+ * @method sendMail
+ * @name Parse.Cloud.sendEmail
+ * @param {Object} data The object of the mail data to send
+ */
+ParseCloud.sendEmail = function (data) {
+  const config = Config.get(Parse.applicationId) || {};
+  const emailAdapter = config.userController.adapter;
```

Comment:
Why `|| {}` - can `config` really be `undefined`?
Also, if config is {}, then `config.userController.adapter` will throw.